CONTRIBUTING.md

```diff
@@ -260,3 +260,20 @@ which combinations are tested regularly.
  * `CREATE_GRAPHVIZ`: if set to `yes`, use `CMake` to generate a dependency
    graph of each target. This is useful when troubleshooting dependencies, or
    simply when trying to document them.
+
+### Upload generated document to personal github pages
+
+If you are using your personal fork as the git origin, there is an easy way to
+upload generated doxygen document to your personal github pages. The following
+command will upload the document under
```

Comment:
s/the document/the documentation/ maybe?

Reply:
Done, thanks!